src/twisted/web/test/test_flatten.py

```diff
@@ -431,6 +431,18 @@ def test_serializeDeferredSlots(self):
         t = tags.p(slot('test'))
         t.fillSlots(test=succeed(tags.em('four>')))
         return self.assertFlattensTo(t, b'<p><em>four&gt;</em></p>')
+    
+    
+    def test_slotDataPropagation(self):
+        """ Test that verify if a slot data inserted in a node is correctly propagated
```

Comment:
See this for how to write docstrings https://jml.io/pages/test-docstrings.html

See the docstring from for the other tests.

No need for "test that", "check that". All tests are written to test something :)

What means "correctly"?

Reply:
I tried to be more explicit !